viadot/sources/databricks.py

```diff
@@ -280,6 +285,10 @@ def create_table_from_pandas(
                 Defaults to 'fail'.
             snakecase_column_names (bool, optional): Whether to convert column names to snake case.
                 Defaults to True.
+            cast_df_columns (bool, optional): Converts column types in DataFrame using utils._cast_df_cols().
+                Object -> string, bool -> Int64 and standardizes the date using the format "%Y-%m-%d %H:%M:%S+00:00".
+                The object type can contain int, string, bool, etc. values, which can cause errors when sending data to Databricks.
+                Defaults to True.
```

Comment:
No need to list what types exactly you're mapping as this can be checked in the util function's docstring. The last sentence is the relevant bit, ie. this param exists because of possible errors with object cols.

Reply:
[📝 Updated docstings and added entry to CHANGELOG.md](https://github.com/dyvenia/viadot/pull/701/commits/b8ec2278fff1a3cbb2c17cdb02c744662261fce7)